src/Microsoft.DotNet.Wpf/src/PresentationFramework/PresentationFramework.csproj

```diff
@@ -9,12 +9,23 @@
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
     <Platforms>AnyCPU;x64;arm64</Platforms>
     <GenerateDependencyFile>false</GenerateDependencyFile>
+    <LangVersion>latest</LangVersion>
```

Comment:
💡 `latest` is a moving target and not a real version. Would recommend using a numeric value for build stability across users who work on the project.